README.rst

```diff
@@ -4,18 +4,7 @@ mpi-sppy
 Optimization under uncertainty for `Pyomo <https://pyomo.org>`_ models.
 
 `Documentation is available at readthedocs <https://mpi-sppy.readthedocs.io/en/latest/>`_ and
-a technical report is on `OOL <http://www.optimization-online.org/DB_HTML/2020/11/8088.html>`_
-
-NOTICE
-^^^^^^
-
-There was a disruptive change on August 11, 2022 concerning how
-options are accessed. See the file ``disruptions.txt`` for more
-information. If you are a new user, this will not affect you,
-regardless of how you install. If you are an
-existing user, you should consider the disruption before updating to
-the latest mpi-sppy. The documentation on readthedocs
-probably refers to the newest version.
+a there is a `paper <https://rdcu.be/dB11w>`_
```

Comment:
Would it be better to use a direct link?

https://link.springer.com/article/10.1007/s12532-023-00247-3